prelude/prelude.purs

```diff
@@ -159,9 +192,34 @@ module Prelude
 
   infixl 4 <*>
 
+  -- | Applys are intuitively Applicatives less `pure`, and more formally a strong lax semi-
```

Comment:
Another thought is that we can always link to the online book in comments for more details if the user wants them.